app/src/main/java/org/wikipedia/dataclient/Service.java

```diff
@@ -263,7 +263,7 @@
     @NonNull Observable<Entities> getWikidataLabels(@Query("ids") @NonNull String idList,
                                                     @Query("languages") @NonNull String langList);
 
-    @GET(MW_API_PREFIX + "action=wbgetclaims")
+    @GET(MW_API_PREFIX + "action=wbgetclaims&property=P180")
     @NonNull Observable<Claims> getClaims(@Query("entity") @NonNull String entity);
```

Comment:
Since this function is called `getClaims()`, I would expect it to return all the claims, and I would be confused if it implicitly filters out certain claims.  It would be much more intuitive if we added a `property` parameter to this function, so that the caller can specify which claim(s) they want to get.